app/code/Magento/Inventory/Model/Indexer/StockItem/Dimension.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\Indexer\StockItem;
+
+/**
+ * Stock Item Dimension
+ */
+class Dimension extends \Magento\Framework\Search\AbstractKeyValuePair
```

Comment:
we already have 
\Magento\Framework\Search\Request\Dimension
why do we create own Dimension object?